src/pages/home/getting-started/configure-datastreams.md

```diff
@@ -46,4 +46,174 @@ The following steps illustrate how to create a sample datastream for a mobile im
 
 Your datastream is ready to be used now. If different settings are required per environment, create new datastreams with the respective configuration settings.
 
-For additional information on datastreams in the Data Collection UI, see the [configuring a datastream guide](https://experienceleague.adobe.com/docs/platform-learn/implement-mobile-sdk/initial-configuration/create-datastream.html).
+## Sending overrides to the Edge Network
```

Comment:
- Since this is an optional configuration, can we extract this out of the getting started section in a separate tutorial?
I recommend putting it under https://developer.adobe.com/client-sdks/edge/edge-network/tutorials/ 

- Our documentation does not cover the set up for the config overrides, can we start the document by mentioning that is a prerequisite and pointing to the ExL datastreams configuration for the setup

Reply:
Got it! I wasn't sure exactly what was the best place to put this info - which is why I put it here. I'll move it to the appropriate location.

Reply:
@calebk1 should we extract this in a separate tutorial file or should we rephrase the title/introduction in the existing file? The reason I am asking is because the original tutorial was only for implementing config overrides with Launch rules in the UI, while this other one you added is for implementing it in the app code client-side. Both achieve the same thing, but they are two different approaches.

If you think we should keep it in the same file, I would recommend moving the new section on top and also rename the file in the left nav and probably the link too. In order to avoid a new link forwarding I am more inclined into keeping them as separate tutorials.